ios/Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -106,15 +109,16 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     }
 
     private func loadEditor() {
-        if let editorURL = editorURL ?? ProcessInfo.processInfo.environment["GUTENBERG_EDITOR_URL"].flatMap(URL.init) {
+        // If you want to load something that's not a manifest, specify the environment variable
+        if let editorURL = ProcessInfo.processInfo.environment["GUTENBERG_EDITOR_URL"].flatMap(URL.init) {
```

Comment:
Unlikely a priority to start, but I imagine we'll need a way to load a fetched asset set/manifest _and_ a local running server for the editor UI bundle. That would allow iterating on the editor UI while relying upon site-specific WordPress packages.

Reply:
Ah yeah I didn't consider this case. If you have a fetched asset set, is there any local editor code running? I think I assumed that the remote server's code would completely replace anything running on a dev server